internal/tools/aws/client.go

```diff
@@ -51,6 +75,21 @@ type api interface {
 	listTagsForCertificate(*acm.ACM, *acm.ListTagsForCertificateInput) (*acm.ListTagsForCertificateOutput, error)
 }
 
+// NewAWSClient returns a new AWS client.
+func NewAWSClient(sess *session.Session) *Client {
+	return &Client{
+		api: &apiInterface{},
+
+		acm:            acm.New(sess),
+		ec2:            ec2.New(sess),
+		iam:            iam.New(sess),
+		rds:            rds.New(sess),
+		s3:             s3.New(sess),
+		route53:        route53.New(sess),
+		secretsManager: secretsmanager.New(sess),
```

Comment:
This could be done more efficiently if we could use construct at first time only. However, if I remember it correctly, there as general concern around session that should be able to pick up changes to the environment. If we can modify session in only one place, we could potentially use a singleton pattern for this client. Session is concurrent-safe for reading but not for writing.